### DISCUSSION TITLE
Media library without Google Play services

### DISCUSSION THREAD

**Discussion by aswedaw** [gateship-one/odyssey#227] [General]
On China ROM MIUI 12.5.1,all music players
, including Odyssey, don't work. Odyssey doesn't start at all. Other players
Shuttle+, Music, Vinyl, Odeon and other Music apps can't scan music files. Why don't music players on F-Droid, scan music and play?

**Comment by gnome17** [comment]
We don't rely on the Google Play services and we are not aware that the media database won't work without it.

What do you mean Odyssey doesn't start? Please provide a log so that we can have a look whats going on there.

**Reply by aswedaw** [reply]
Music Players including **Odyssey, 
Music,Vinyl and Shuttle** don't work
on MIUI 12.5.1 China ROM phones.
